components/Page.js

```diff
@@ -3,6 +3,7 @@ import { bindActionCreators } from 'redux'
 import Cookies from 'next-cookies'
 import LocalForage from 'localforage'
 import React from 'react'
+import Router from 'next/router'
```

Comment:
Router should be imported from routes.js instead of next/router.

Reply:
Good catch. I'll fix that.